kuma/contributions/jinja2/contributions/contribute.html

```diff
@@ -0,0 +1,228 @@
+{% extends "base.html" %}
+
+
+{% block body_attributes %}id="contribute"{% endblock %}
+
+{% block title %}{{ page_title(_('Contribute')) }}{% endblock %}
+
+{% block site_css %}
+    {{ super() }}
+    {% stylesheet 'jquery-ui' %}
+{% endblock %}
+
+{% block site_js %}
+    {{ super() }}
+    <script src="https://checkout.stripe.com/checkout.js"></script>
+    {% javascript 'jquery-ui' %}
+    {% javascript 'contribute' %}
+{% endblock %}
+
+{% block extrahead %}
+    {# TODO: Implement copy & design for the share functionality #}
+    {% set seo_description = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud.' %}
+    {% set social_title = _('Support MDN') %}
+    <meta property="og:description" content="{{ seo_description }}">
+    <meta name="description" content="{{ seo_description }}">
+    <meta name="twitter:description" content="{{ seo_description }}">
+    <meta name="twitter:title" content="{{ social_title }}">
+    <meta property="og:title" content="{{ social_title }}">
+{% endblock %}
+
+{% block document_head %}
+    {% with form=form, is_popover=False %}
+        {% include "contributions/includes/contribution-form.html" %}
+    {% endwith %}
+{% endblock %}
+
+{% block contrib_popup %}{% endblock %}
+
+{% block content %}
+
+<div id="contributions-page" class="conributions-page">
+    <div class="section">
+        <img src="{{ static('img/hero-dino-blank.png') }}" class="backdrop-image" alt="" role="presentation">
+        <div class="index">
+            <h2>{{_('FAQs')}}</h2>
+        </div>
+        <ul class="faqs clear">
+            <li class="faq">
+                <h3>{{_('Why is MDN asking me for money?')}}</h3>
+                <span class="faq-number">1</span>
```

Comment:
Would it be better to use an ordered list instead? You can still have custom treatment but, from an accessibility point of view, the list will be treated as an ordered list, and it makes more semantic sense.

You can then get rid of the `span` elements, and generate the numbers using a combination of `h3:before` and CSS counters(https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Lists_and_Counters/Using_CSS_counters) that can be styled as you wish.

Reply:
Fixed in latest PR.